### PR TITLE
Refine corpus

### PR DESCRIPTION
**Pull request by bioball**
This removes snippet tests from test/corpus, because they were contributing more noise than useful information; it's not practical to review those tests because there's so much code being introduced.

This also adds some more tests that were previously only covered by snippet tests.

### CODE REVIEW

File: test/corpus/object/objectMember.txt [hunk 1]
```diff
@@ -13,10 +13,10 @@ res4 {1;2;3;}
 res5{1}
 res6{1;2}
 res7{foo = 1}
-res8{foo = 1 bar = 2}
+res8{foo = 1; bar = 2}
 res9{nested{1}}
-res10{["foo"] = 1 ["bar"] = 2}
-res11 { new {} new {} new {} }
+res10{["foo"] = 1; ["bar"] = 2}
+res11 { new {}; new {}; new {} }
```
Comment (holzensp) [inline test/corpus/object/objectMember.txt:19]: Keeping some negative (`ERROR`) cases in might be valuable too (considering allowing non-semi-colon-ed same-line object literals is explicitly undesirable)

File: test/corpus/string/multiLineInterpolation.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ================================================================================
-multiline
+string/multilineInterpolation
```
Comment (holzensp) [inline test/corpus/string/multiLineInterpolation.txt:2]:
```suggestion
string/multiLineInterpolation
```

(or rename the file)

**Review by stackoverflow — APPROVED** [review]

File: test/corpus/basic/comments.txt [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
+========
+comments
+========
 
 x = 10 // end-of-line comment
 
 y = 20 /*
 multi
 line
-/* nested */
```
[the remark this answers is not in the document]
Comment (bioball) [reply test/corpus/basic/comments.txt:10]: Thanks! Filed some more issues that I found yesterday too.

File: test/corpus/object/objectMember.txt (hunk 1 above)
Comment (bioball) [reply test/corpus/object/objectMember.txt:19]: It's actually a bug that it's being parsed this way.

But: only pkl-core parses this correctly; both pkl-intellij and tree-sitter-pkl generate incorrect parse trees here.

File: test/corpus/string/multiLineInterpolation.txt (hunk 2 above)
Comment (bioball) [reply test/corpus/string/multiLineInterpolation.txt:2]: The `string/` prefix is a little redundant, no? These tests are hierarchical and tree-sitter tells you which test this belongs in.

E.g. from their test report:

```
  basic:
    annotation:
        1. ✓ annotation
    comments:
        2. ✓ comments
    types:
        3. ✓ types
  class:
    constModifier:
        4. ✓ constModifier
    fixedModifier:
        5. ✓ fixedModifier
```

Comment (bioball) [reply test/corpus/string/multiLineInterpolation.txt]: Argh, I did this but of course GIt had trouble tracking this thanks to case insensitive file systems.

**Pull request merged**